src/Controller/AnnouncementController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\AnnouncementManager;
+
+class AnnouncementController extends AbstractController
+{
+    /**
+     * List items
+     */
+    public function index(): string
+    {
+        return $this->twig->render('Announcement/index.html.twig');
+    }
+
+    /**
+     * Show cart for a category
+     */
+    public function category(): string
```

Comment:
category relates to ?

Reply:
@Kengaroo can you answer this ?

Reply:
Announcements page. 
Change please function name to 'index()' and in routes.php to  'announcement' => ['AnnouncementController', 'index',],
Thanks.